include/attribute.h

```diff
@@ -26,4 +26,10 @@
 # define TCB_NONNULL(params)
 #endif
 
+#if TCB_GNUC_PREREQ(7,1)
+# define TCB_FALLTHROUGH __attribute__((fallthrough));
```

Comment:
Oh, does `__attribute__((fallthrough))` require specifically gcc 7.1+? I ask because in the LKRG project I lazily put a 7+ check, not 7.1+, so maybe we need to revise it there. Is there a quick way to check which gcc version first introduced which attribute?

Reply:
Unfortunately there is no way to check for that, but browsing the manuals.  :/  In this case 7+ will work, as there was no gcc v7.0 release; the first release in the gcc-7 series was v7.1.0.

Anyways, I've dropped this in my recent rebase, as there are easier ways to get the result, we want here.